markdown/appendices/resources.md

```diff
@@ -25,10 +25,25 @@ title: Other Doomsday Resources
 
 - [Legacy Breakfast](https://legacy-breakfast.com/tag/doomsday/)
 - [Eternal Central](https://www.eternalcentral.com/tag/doomsday/)
+- [The Dead Format episode 106 ft. Nevilshute](https://soundcloud.com/the-dead-format/episode-106-doomsday-with-nevilshute-martin-nielsen)
 
 ## Videos
 
+- [DK_TTGL runs a League with Grixis Meandeck (June 2020) - Portugese]
```

Comment:
@Doishy Unsure it's very relevant to add non-english content 🤔

Reply:
I would still like it added, it's only a small link but it is a well made video and we are trying to be at least somewhat international :)

Reply:
Works for me 🙌